docs/concepts/why-filecoin.md

```diff
@@ -0,0 +1,79 @@
+
+## Self-healing
+
+The Filecoin network continually verifies that files are stored correctly. The Filecoin blockchain has a built-in self-healing process where faulty miners are detected and their files are redistributed to reliable miners.
+
+## Verifiable traces
+
+In the process of self-healing, Filecoin generates verifiable traces that files have been stored correctly over time. Clients can efficiently scan these traces to confirm that their files have been stored correctly, even if the client was offline at the time. Any observer can check any miner’s track record and will notice if the miner has been faulty or offline in the past.
+
+## Reputation not marketing
+
+In Filecoin, storage providers prove their reliability through their track record published on the blockchain, not through marketing claims published by the provider themselves. Users don’t need to rely on status pages or self-reported statistics from storage providers.
+
+## Choice of tradeoffs
+
+Users get to choose their own tradeoffs between cost, redundancy and speed. Users are not limited to a set list of data centres offered by their provider, but can choose to store their files on any miner participating in Filecoin.
+
+## Puts empty storage to use
+
+Filecoin puts disk space that would otherwise sit empty to use. Miners are incentivised to join Filecoin because they get paid for their participation, unlike other distributed networks where there is no economic incentive. Filecoin also makes it easy to join because miners self-manage their storage, earning Filecoin tokens without needing human supervision.
+
+## Censorship resistance
+
+Filecoin resists censorship because there is no central provider that can be coerced into deleting files or withholding service. The network is made up of many different computers run by many different people and organizations. Faulty or malicious actors are noticed by the network and removed automatically.
+
+## Useful blockchain
+
+In Filecoin, miners are rewarded for providing storage, not for performing wasteful computations. Filecoin secures its blockchain using proof of file replication and proof of storage over time. It doesn’t rely on energy intensive proof-of-work schemes like other blockchains. Miners are incentivised to amass hard drives and put them to use by storing files. Filecoin doesn’t incentivise hoarding of graphics cards or application-specific integrated circuits for the sole purpose of mining.
+
+## Provides storage to other blockchains
+
+Filecoin excels at storing large files, whereas other blockchains can typically only store very small amounts of data, very expensively. Filecoin can provide its storage to other blockchains, allowing them to store large files. In the future, mechanisms will be added to Filecoin enabling transactions on Filecoin’s blockchain to interoperate with transactions on other blockchains.
```

Comment:
Part of this paragraph suggests Filecoin IS a blockchain and part suggests it USES a blockchain. I assume the latter is more accurate, and elsewhere we're referring to it as a network.

Reply:
Thanks! I have reworded this part to make the difference clearer.